src/modules/mavlink/mavlink_messages.cpp

```diff
@@ -541,11 +541,11 @@ class MavlinkStreamSysStatus : public MavlinkStream
 private:
 	MavlinkOrbSubscription *_status_sub;
 	MavlinkOrbSubscription *_cpuload_sub;
-	MavlinkOrbSubscription *_battery_status_sub;
+	MavlinkOrbSubscription *_battery_status_sub[BOARD_NUMBER_BRICKS];
 
 	uint64_t _status_timestamp{0};
 	uint64_t _cpuload_timestamp{0};
-	uint64_t _battery_status_timestamp{0};
+	uint64_t _battery_status_timestamp[BOARD_NUMBER_BRICKS];
```

Comment:
Could you initialize it with {0} like the ones above?

Reply:
I have added that initialization to the constructor.

Reply:
Aha ok.